rest-happstack/src/Rest/Driver/Happstack/Docs.hs

```diff
@@ -5,15 +5,21 @@ import Control.Monad
 import Control.Monad.Trans
 import Happstack.Server
 import Rest.Api
+
 import Rest.Gen.Base
 import Rest.Gen.Docs
 
 -- | Web interface for documentation
 apiDocsHandler :: (ServerMonad m, MonadPlus m, FilterMonad Response m, MonadIO m) => String -> String -> Api a -> m Response
-apiDocsHandler rootURL tmpls api =
```

Comment:
Did anything change here? It's hard to see with the change from `let .. in` to `where`...